Python/Product/Cookiecutter/Shared/Interpreters/CPythonInterpreterFactoryConstants.cs

```diff
@@ -0,0 +1,64 @@
+// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.CookiecutterTools.Interpreters {
```

Comment:
>Interpreters [](start = 38, length = 12)

Let's plan to aggressively simplify this namespace so we can avoid duplicating too much code (e.g. I'd rather leave out PythonLanguageVersion.cs, as it requires modification to avoid throwing errors when new Python versions come out)